source/tomopy/recon/rotation.py

```diff
@@ -203,14 +204,14 @@ def _find_center_cost(
 
 
 def find_center_vo(tomo, ind=None, smin=-50, smax=50, srad=6, step=0.25,
-                   ratio=0.5, drop=20):
+                   ratio=0.5, drop=20, parallel=True):
```

Comment:
Please match the rest of the library by using `ncore, nchunk` parameters with default value of `None`. The `distribute_jobs` function automatically chooses values for these parameters.

https://github.com/tomopy/tomopy/blob/1047c0b59de3bf5026209b670ba4b1ebd03ba9f9/source/tomopy/prep/stripe.py#L88-L90